src/lib/mongoBackend/connectionOperations.cpp

```diff
@@ -359,6 +377,7 @@ bool collectionUpdate
   {
     connection->update(col.c_str(), q, doc, upsert);
     releaseMongoConnection(connection);
+    TIME_STAT_MONGO_WRITE_WAIT_STOP();
```

Comment:
Fixed in e99e0e4.
As now the 'partial' measurements of "last" is accumulated until 'request completed' I understand this must be taken care of.